### PR TITLE
Add SPLUNK_PROFILER_CALL_STACK_INTERVAL

### PR DESCRIPTION
**Pull request by breedx-splk**
This is a remake of #150.

I thought it also made sense to make a default column, and the oddly centered content looked odd, so I just made it left aligned.

### CODE REVIEW

File: specification/configuration.md [hunk 1]
```diff
@@ -158,14 +158,15 @@ For all use-cases that support environment variables (e.g. applications and
...
-| `SPLUNK_PROFILER_ENABLED` (`false`)             | Whether profiling is enabled. [2]                                               |
-| `SPLUNK_PROFILER_LOGS_ENDPOINT` ()              | Where profiling data is sent. By default `OTLP_EXPORTER_OTLP_ENDPOINT` is used. |
-| `SPLUNK_REALM` (`none`)                         | Which realm to send exported data. [3]                                          |
-| `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` (`true`) | Whether `Server-Timing` header is added to HTTP responses. [4]                  |
+| Name                                   | Default | Description                                                     |
+|----------------------------------------|---------|-----------------------------------------------------------------|
+| `SPLUNK_ACCESS_TOKEN`                  | none    | Access token added to exported data. [1]                        |
+| `SPLUNK_METRICS_ENDPOINT`              | none    | Endpoint for metrics data ingest.                               |
+| `SPLUNK_PROFILER_CALL_STACK_INTERVAL`  | 10s     | Interval at which call stacks are sampled                       |
```
Comment (flands) [inline specification/configuration.md:165]: nit

```suggestion
| `SPLUNK_PROFILER_CALL_STACK_INTERVAL`  | 10s     | Interval at which call stacks are sampled.                      |
```

File: specification/configuration.md [hunk 2]
```diff
@@ -158,14 +158,15 @@ For all use-cases that support environment variables (e.g. applications and
...
+| `SPLUNK_PROFILER_ENABLED`              | false   | Whether profiling is enabled. [2]                               |
+| `SPLUNK_PROFILER_LOGS_ENDPOINT`        | *       | Where profiling data is sent. Defaults to the value in `OTLP_EXPORTER_OTLP_ENDPOINT`  |
+| `SPLUNK_REALM`                         | `none`  | Which realm to send exported data. [3]                          |
```
Comment (flands) [inline specification/configuration.md:168]: What's the difference between a ticked none and a not ticked none? Perhaps blank default should not list anything in this column? Or I guess actual values can be ticked?

File: specification/configuration.md (hunk 1 above)
Comment (breedx-splk) [reply specification/configuration.md:165]: I never put periods at the end of partial sentence fragments. 🤓

File: specification/configuration.md (hunk 2 above)
Comment (breedx-splk) [reply specification/configuration.md:168]: I agree, this is confusing. I didn't actually know the answer and just brought it forward the way it was. Lemme go find the answer...

Comment (pellared) [reply specification/configuration.md:168]: `none` is a valid and expected default value for the `SPLUNK_REALM`. It was discussed here: https://github.com/signalfx/gdi-specification/pull/148

The other values should be left blank

File: specification/configuration.md [hunk 3]
```diff
@@ -158,14 +158,15 @@ For all use-cases that support environment variables (e.g. applications and
...
-| `SPLUNK_PROFILER_ENABLED` (`false`)             | Whether profiling is enabled. [2]                                               |
-| `SPLUNK_PROFILER_LOGS_ENDPOINT` ()              | Where profiling data is sent. By default `OTLP_EXPORTER_OTLP_ENDPOINT` is used. |
-| `SPLUNK_REALM` (`none`)                         | Which realm to send exported data. [3]                                          |
-| `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` (`true`) | Whether `Server-Timing` header is added to HTTP responses. [4]                  |
+| Name                                   | Default | Description                                                     |
+|----------------------------------------|---------|-----------------------------------------------------------------|
+| `SPLUNK_ACCESS_TOKEN`                  | none    | Access token added to exported data. [1]                        |
+| `SPLUNK_METRICS_ENDPOINT`              | none    | Endpoint for metrics data ingest.                               |
+| `SPLUNK_PROFILER_CALL_STACK_INTERVAL`  | 10s     | Interval at which call stacks are sampled.                      |
```
Comment (pellared) [inline specification/configuration.md:165]: I suggest changing it to `10000` as milliseconds is the only required (and default) duration unit. See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#duration

File: specification/configuration.md (hunk 2 above)
Comment (breedx-splk) [reply specification/configuration.md:168]: Thanks for the clarification, @pellared I'll get it changed.

File: specification/configuration.md (hunk 3 above)
Comment (breedx-splk) [reply specification/configuration.md]: Ok sure, we had a discussion in a previous PR about units, but I'm fine changing it to ms.

Comment (pellared) [reply specification/configuration.md]: We can support units like `ms` but it is safer to use the the "required" unit in specs 😉

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
I am just wondering if we can add new settings to "stable" Instrumentation Libraries configuration for a behavior that is "experimental" (Profiling)

**Comment by flands** [comment]
> I am just wondering if we can add new settings to "stable" Instrumentation Libraries configuration for a behavior that is "experimental" (Profiling)

Oh, that is fun... @breedx-splk thoughts on this? Maybe we need a column to donate? Or are you comfortable marking stable?

**Comment by breedx-splk** [comment]
> > I am just wondering if we can add new settings to "stable" Instrumentation Libraries configuration for a behavior that is "experimental" (Profiling)
> 
> Oh, that is fun... @breedx-splk thoughts on this? Maybe we need a column to donate? Or are you comfortable marking stable?

I think it's perfectly fine to add experimental features to stable components. Since `profiling.md` was assimilated I guess we did lose some tracking of what's experimental and what's stable. 

A column could help mitigate, but fear that the simple table (which I've already made more complicated by adding defaults) to turn into a convoluted stability matrix.

So far, there's really just one category of stuff that's experimental. Can we handle that with a footnote or something?

**Comment by flands** [comment]
> So far, there's really just one category of stuff that's experimental. Can we handle that with a footnote or something?

Fine by me

**Review by kubawach — APPROVED** [review]
LGTM

**Comment by kubawach** [comment]
> > So far, there's really just one category of stuff that's experimental. Can we handle that with a footnote or something?
> 
> Fine by me

@breedx-splk going to add the footnote? I would merge this ASAP :)

**Pull request merged**